tests/transformations/loop_to_map_test.py

```diff
@@ -66,8 +68,8 @@ def make_sdfg(with_wcr, map_in_guard, reverse_loop, use_variable, assign_after):
                                 """\
 static std::mutex mutex;
 std::unique_lock<std::mutex> lock(mutex);
-std::ofstream of("loop_to_map_test.txt", std::ofstream::app);
-of << i << "\\n";""",
+std::ofstream of("{}", std::ofstream::app);
```

Comment:
maybe fstring here?

Reply:
Done in 43cc83a7c74d7625c20fbe108e65e6ba32ea7ea9